src/components/tutorial-card/helpers/with-bookmark.tsx

```diff
@@ -0,0 +1,25 @@
+import { AUTH_ENABLED } from 'hooks/use-authentication'
+import { TutorialCardBookmarkButton } from 'components/bookmark-button'
+import TutorialCard from '..'
+import { TutorialCardProps } from '..'
+
+interface TutorialCardWithBookmarkProps extends TutorialCardProps {
+	isBookmarked: boolean
+}
+
+export function TutorialCardWithBookmark(props: TutorialCardWithBookmarkProps) {
```

Comment:
This is a tutorial card with the eyebrowSlot composed with the connected bookmark. Eventually we will have many different types of eyebrow components to render in this slot, once we implement progress.